src/core/qgstiles.cpp

```diff
@@ -16,22 +16,65 @@
 #include "qgstiles.h"
 
 #include "qgslogger.h"
+#include "qgscoordinatereferencesystem.h"
+#include "qgssettings.h"
 
-QgsTileMatrix QgsTileMatrix::fromWebMercator( int zoomLevel )
+QgsTileMatrix QgsTileMatrix::fromWebMercator( const int &zoomLevel )
```

Comment:
Is there a reason to change from `int` to `const int &` ? For basic numeric values it makes more sense to pass them by value (`int`) instead of const reference. (Same for int+double arguments in fromCustomDef())